spec/project/changelog_spec.rb

```diff
@@ -69,7 +69,7 @@
       end
 
       it 'ends with a punctuation' do
-        expect(bodies).to all(match(/[\.\!]$/))
+        expect(bodies).to all(match(/[.!]$/))
```

Comment:
We could probably use `end_with` instead of `match`.

Reply:
@bquorning since there is a `end_with` matcher which doesn't support checking that a string ends with either one of the characters, this would become
```diff
-        expect(bodies).to all(match(/[.!]$/))
+       expect(bodies).to all(end_with('.').or(end_with('!')))
```
not sure if it's better though

Reply:
Nah, I was hoping for `end_with('.', '!')`. 

Let’s just leave it as it is.

Reply:
Yeah, but that actually requires ending with `.!` ;)